AwesomeCalculator/app/src/main/java/com/example/drcreeper/awesomecalculator/asynktasks/GetOperationsListAsyncTask.java

```diff
@@ -0,0 +1,43 @@
+package com.example.drcreeper.awesomecalculator.asynktasks;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.os.AsyncTask;
+
+import com.example.drcreeper.awesomecalculator.historywriter.HistoryDatabaseOpenHelper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GetOperationsListAsyncTask extends AsyncTask<Void,Void,List<String>> {
+
+    Context context;
+
+    public Context getContext() {
+        return context;
+    }
+
+    public void setContext(Context context) {
+        this.context = context;
+    }
+
+    @Override
+    protected List<String> doInBackground(Void... voids) {
```

Comment:
Convert rows from database not just into string but in POJO objects. Also please update database scheme to include next columns instead of column "solve": "operand_1", "operand_2", "operator", "result". Also add "timestamp" field. Write migration script from old DB scheme to new one so that no data will be lost and reasonable defaults be provided for new field. Read about timestamp here: https://www.unixtimestamp.com/